lib/src/base-styles/_nx-card.scss

```diff
@@ -107,75 +78,64 @@
   }
 
   .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
+    align-self: center;
+    grid-area: card-content;
     margin: 0;
     padding: 0;
+    row-gap: 0;
   }
 
   .nx-card__call-out {
-    -ms-grid-row-align: center;
     align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 1;
     grid-area: card-callout;
-    -ms-grid-column-align: center;
     justify-self: center;
 
     align-items: center;
     border-right: $nx-border-default;
     display: flex;
     height: 100%;
-    margin: 0 $nx-spacing-xxl 0 0;
-    padding: 0 $nx-spacing-xxl 0 0;
+    margin: 0 $nx-spacing-l 0 0;
+    padding: 0 $nx-spacing-l 0 0;
   }
 
-  .nx-card__footer {
+  .nx-card__actions {
     box-sizing: content-box;
-    -ms-grid-row-align: center;
     align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 3;
-    grid-area: card-footer;
-    -ms-grid-column-align: start;
+    grid-area: card-actions;
     justify-self: start;
 
     @include container-horizontal;
     align-items: center;
-    border: none;
     border-left: $nx-border-default;
     display: flex;
     height: 100%;
-    margin: 0 0 0 $nx-spacing-xxl;
-    padding: 0 0 0 $nx-spacing-xxl;
+    margin: 0 0 0 $nx-spacing-l;
+    padding: 0 0 0 $nx-spacing-l;
     // necessary to make the link wrap
     max-width: 150px;
   }
 }
 
-// Styles for the alternate row layout
-.nx-card-container--row {
-  .nx-card__content-container {
-    align-items: center;
-    display: flex;
+// Common card Helpers
+.nx-card__call-out {
+  @include bold();
+
+  font-size: 32px;
+
+  .nx-icon--card-call-out {
+    font-size: 48px;
+  }
+
+  .nx-icon--card-call-out-xl {
+    font-size: 64px;
+  }
+}
+
+.nx-card__content--2-column {
```

Comment:
If `.nx-card__content` was flex-based this could probably be as simple as overriding the `flex-direction` with no other changes

Reply:
> If `.nx-card__content` was flex-based this could probably be as simple as overriding the `flex-direction` with no other changes

It wasn't quite that easy, but yeah: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/1a904c393841bca80421915a60e0b8e1d7f4305f

Reply:
> It wasn't quite that easy, but yeah: 1a904c3

It could be made that easy.  The `align-items: center` here could (and maybe even should) be moved to `.nx-card__content`.  And then if `.nx-card__content` specified `gap` instead of `row-gap`, we could get rid of `column-gap` here.

Reply:
> > It wasn't quite that easy, but yeah: [1a904c3](https://github.com/sonatype/sonatype-react-shared-components/commit/1a904c393841bca80421915a60e0b8e1d7f4305f)
> 
> It could be made that easy. The `align-items: center` here could (and maybe even should) be moved to `.nx-card__content`. And then if `.nx-card__content` specified `gap` instead of `row-gap`, we could get rid of `column-gap` here.

That does work: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/c8ff12e1387070f401c2c4f332b097c233bd3fbc